oop/video_games/borderlands.py

```diff
@@ -0,0 +1,20 @@
+from oop.video_games.fps import FPS
+
+class Borderlands(FPS):
+    def __init__(self, title: str, multiplayer: bool):
+        super().__init__(title, multiplayer)
+        self.title = title
+        self.multiplayer = multiplayer
+        self.arena = []
+    # a skag (one type of enemy) spawns in the arena
+    def spawn_enemy(self, enemy_type: str):
+        skag = Borderlands(enemy_type, multiplayer = False)
```

Comment:
I would caution you against creating an instance of Borderlands for 'skag' and adding it to the arena. the correct implementation would be to have an enemy class, but that was beyond the scope of this assignment.